pry-stack_explorer.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.specification_version = 4
 
-  s.add_runtime_dependency 'binding_of_caller', '~> 0.7'
+  s.add_runtime_dependency 'binding_of_caller', '~> 1.0'
```

Comment:
After upgrading `binding._of_caller`, I confirmed all the test passed green, at least [at my end](https://travis-ci.org/github/fursich/pry-stack_explorer)